### PR TITLE
libprelude: new formula

### PR DESCRIPTION
**Pull request by maxim-belkin**
- [x] Have you followed the [guidelines for contributing](https://github.com/Linuxbrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Linuxbrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

new formula.

### CODE REVIEW

**Comment by sjackman** [comment]
What's the purpose of this formula? Does another `core` formula depend on it?

**Comment by rwhogg** [comment]
As explained by @maxim-belkin on #1186, this is a possible dependency for PAM.

File: Formula/libprelude.rb [hunk 1]
```diff
@@ -4,38 +4,44 @@ class Libprelude < Formula
   url "https://www.prelude-siem.org/attachments/download/721/libprelude-3.1.0.tar.gz"
   sha256 "b8fbaaa1f2536bd54a7f69fe905ac84d936435962c8fc9de67b2f2b375c7ac96"
 
+  option "without-ruby", "Build without Ruby bindings"
```
Comment (sjackman) [inline Formula/libprelude.rb]: I see now that you've included options for all of the languages supported by `libprelude`. That seems pretty reasonable to me.

**Comment by sjackman** [comment]
Thanks for this formula, Maxim!

**Comment by maxim-belkin** [comment]
Thanks, Shaun. Unfortunately, the bottle that CircleCI built for this formula is not relocatable because of strings in dynamic libraries. I think this might be a non-desired feature (bug) in their code.

**Comment by sjackman** [comment]
You can open an upstream issue if you'd like to pursue it further. There's lots of non-relocatable bottles though, roughly one third are relocatable, as I recall.

**Pull request merged**